graphics/ic_launcher_monochrome_sc.xml

```diff
@@ -0,0 +1,39 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This is just a duplicate of the default one that lives within `main` resources. I followed the existing pattern of having this be in the `graphics` folder, so you should be able to switch back from alternate builds to the main one, but I didn't test that flow.